Packs/Gem/Integrations/Gem/Gem.yml

```
@@ -0,0 +1,1059 @@
+category: Cloud Services
+sectionOrder:
+  - Connect
+  - Collect
+commonfields:
+  id: Gem
+  version: -1
+defaultmapperin: Gem Mapper
+defaultclassifier: Gem Classifier
+configuration:
+  - display: Incident type
+    name: incidentType
+    type: 13
+    defaultvalue: Gem Alert
+    section: Connect
+    required: false
+  - display: API Endpoint
+    defaultvalue: "https://app.gem.security/api"
+    additionalinfo: "The API endpoint to use for connection (US or EU)"
+    name: api_endpoint
+    type: 15
+    section: Connect
+    required: true
+    options:
+      - "https://app.gem.security/api"
+      - "https://eu-west-1.app.gem.security/api"
+  - display: Service Account ID
+    additionalinfo: The Service Account ID to use for connection
+    name: client_id
+    type: 0
+    section: Connect
+    required: true
+  - display: Service Account Secret
+    additionalinfo: The Service Account Secret to use for connection
+    name: client_secret
+    type: 4
+    section: Connect
+    required: true
```

Type 4 is deprecated, please use type 9 instead.
For example: 
```suggestion
  -  display: Service Account ID
     name: credentials
    defaultvalue: ""
    type: 9
    required: true
    section: Connect
    displaypassword: Service Account Secret
```